SignalServiceKit/src/Contacts/TSThread.h

```diff
@@ -11,6 +11,9 @@ NS_ASSUME_NONNULL_BEGIN
 @class TSInvalidIdentityKeyReceivingErrorMessage;
 
 typedef NSString *ConversationColorName NS_STRING_ENUM;
+
+ConversationColorName ConversationColorNameForString(NSString *value);
```

Comment:
Swift can't convert `ConversationColorName` to `String` and back, so I made this convenience method.

Reply:
Huh... what happens when you use `ConversationColorName(rawValue: colorName)` and (eg) `ConversationColorNameCrimson.rawValue`?

Reply:
Oh nice, thanks.